setup.py

```diff
@@ -28,19 +28,27 @@ def readme():
 ]
 
 spinx_requires = [
-    'sphinx~=2.2;python_version>="3.6"',
+    'sphinx~=3.5;python_version>="3.6"',
+    'jinja2==3.0.3;python_version>="3.6"',
 ]
 
 tests_require = [
-    'freezegun',
+    'freezegun==1.0.0',
     'mock;python_version<"3.3"',
     'mypy~=0.740;python_version>"3.4"',
     'pytest>4.2,<5.4',
-    'pytest-cov',
+    'pytest-cov~=2.5',
+    'coverage~=5.2',
     'pytest-runner',
     'pytz',
+    'importlib-metadata~=5.0;python_version>"3.6"'
 ] + currency_requires + country_requires + spinx_requires
 
+mypy_requires = [
```

Comment:
Added a mypy requires section as a good majority of the initial mypy errors were just complaining that `six` and `pytz` did not have type annotations available.